data/news/multicore/multicore-2020-11.md

```diff
@@ -7,7 +7,7 @@ tags: [multicore]
 
 Welcome to the November 2020 Multicore OCaml report! This update along with the [previous updates](https://discuss.ocaml.org/tag/multicore-monthly) have been compiled by @shakthimaan, @kayceesrk, and @avsm.
 
-**Multicore OCaml:** Since the support for systhreads has been merged last month, many more ecosystem packages compile.  We have been doing bulk builds (using a specialised [opam-health-check instance](http://check.ocamllabs.io:8082)) against the opam repository in order to chase down the last of the lingering build bugs. Most of the breakage is around packages using C stubs related to the garbage collector, although we did find a few actual multicore bugs (related to the thread machinery when using dynlink). The details are under "ecosystem" below. We also spent a lot of time on optimising the stack discipline in the multicore compiler, as part of writing a draft paper on the effect system (more details on that later).
+**Multicore OCaml:** Since the support for systhreads has been merged last month, many more ecosystem packages compile.  We have been doing bulk builds (using a specialised [opam-health-check instance](http://check.ocamllabs.io:8082 - [1 Client error: Couldn't connect to server])) against the opam repository in order to chase down the last of the lingering build bugs. Most of the breakage is around packages using C stubs related to the garbage collector, although we did find a few actual multicore bugs (related to the thread machinery when using dynlink). The details are under "ecosystem" below. We also spent a lot of time on optimising the stack discipline in the multicore compiler, as part of writing a draft paper on the effect system (more details on that later).
```

Comment:
```suggestion
**Multicore OCaml:** Since the support for systhreads has been merged last month, many more ecosystem packages compile.  We have been doing bulk builds (using a specialised [opam-health-check instance](https://check.ci.ocaml.org/)) against the opam repository in order to chase down the last of the lingering build bugs. Most of the breakage is around packages using C stubs related to the garbage collector, although we did find a few actual multicore bugs (related to the thread machinery when using dynlink). The details are under "ecosystem" below. We also spent a lot of time on optimising the stack discipline in the multicore compiler, as part of writing a draft paper on the effect system (more details on that later).
```